app/views/sessions/new.html.erb

```diff
@@ -11,6 +11,11 @@
       <%= f.label :password %>
       <%= f.password_field :password, class: 'form-control' %>
 
+      <%= f.label :remember_me, class: "checkbox inline" do %>
```

Comment:
For the sake of simplicity, I just want to propose an alternative. Why not always assume 'remember' every single time?